common/Android.bp

```diff
@@ -9,6 +9,8 @@ java_library {
 
     srcs: ["src/**/*.java"],
 
+    required: ["privapp-permissions-transmitpower"],
```

Comment:
move to TransPowerSensors/Android.bp and leave the .xml extenstion

Reply:
> move to TransPowerSensors/Android.bp

That means the AOSP build system is fundamentally broken.

> leave the .xml extenstion

Turns out this convention - inherited from `ModemConfig` - needs `filename_from_src: true,`, which was forgotten here.

Reply:
added

Reply:
Wasn't needed, I was just curious why this _untested_ PR wasn't working OOTB :wink: - but thanks!